README.md

```diff
@@ -135,6 +135,22 @@ const didWithCap = did.withCapability(cacao)
 const didWithCap2 = new DID({provider, resolver: KeyResolver.getResolver(), capability: cacao})
 ```
 
+## Security Considerations
+
+Ceramic allows for keys attached to DIDs to continue making updates to streams after revocation for a certain grace period. This is done to ensure that if multiple updates were committed to different nodes before the key was revoked, but the nodes chose to anchor commits at different times, the updates still make it through.
```

Comment:
```suggestion
Ceramic allows for keys attached to DIDs to continue making updates to streams after revocation for a certain grace period. This is done to avoid incorrectly rejecting valid signatures as being performed with a revoked key when in fact the key was valid at the time of the signature.  This can happen due to the implementation details of how Ceramic determines when a DID issued a signature as well as how the 3ID DID method built on Ceramic determines when an authentication key for a 3ID is revoked.  Since we cannot trust the system clock time on the client's machine when performing a signature or key revocation, Ceramic periodically "anchors" updates on a blockchain to get a timestamp before which we know the write must have happened.  Consider the following scenario:
```

Reply:
thank you for these suggestions, very helpful. i need to get better at writing these kind of documents.

Reply:
I don't think we need to explicitly mention 3ID here.